<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMedicalRecord</class>
 <widget class="QWidget" name="FormMedicalRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>613</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>928</width>
    <height>613</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Медицинская карта</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;В АМБУЛАТОРНЫХ УСЛОВИЯХ №&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="card_number">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;МЕДИЦИНСКАЯ КАРТА&lt;/p&gt;&lt;p&gt;ПАЦИЕНТА, ПОЛУЧАЮЩЕГО МЕДИЦИНСКУЮ ПОМОЩЬ&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_02" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>1. Дата заполнения медицинской карты: число</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="creation_date_day">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>месяц</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="creation_date_month">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>год</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="creation_date_year">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_03" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>2. Фамилия, имя, отчество</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fio">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_04" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>3. Пол:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sex">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>4. Дата рождения: число</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="date_birth_day">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>месяц</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="date_birth_month">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>год</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="date_birth_year">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_05" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="2" column="3">
       <widget class="QLineEdit" name="address_house">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>квартира</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="address_area">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="address_apartment">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="address_street">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>город</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>улица</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>5. Место регистрации: субъект Российской Федерации</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>район</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>дом</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>тел.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QLineEdit" name="tel">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QLineEdit" name="address_city">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="5">
       <widget class="QLineEdit" name="address_subject">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>населённый пункт</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6" colspan="2">
       <widget class="QLineEdit" name="address_locality">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_06" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>6. Местность:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="locality_city">
        <property name="text">
         <string>городская - 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="locality_village">
        <property name="text">
         <string>сельская - 2</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_07" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>7. Полис ОМС: серия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="polis_series">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>№</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="polis_number">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>8. СНИЛС</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="snils">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_08" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>9. Наименование страховой медицинской организации</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="medical_organization">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_09" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>10. Код категории льготы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="concession_code">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>11. Документ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="documnet">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>: серия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="documnet_series">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>№</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="documnet_number">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_10" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;11. Семейное положение: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="marital_status_marriage">
        <property name="text">
         <string>состоит в зарегестрированном браке - 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="marital_status_single">
        <property name="text">
         <string>не состоит в браке - 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="marital_status_unknown">
        <property name="text">
         <string>неизвестно - 3</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_11" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_29">
        <property name="text">
         <string>12. Образование: профессиональное:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="education_professional_higher">
        <property name="text">
         <string>высшее - 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="education_professional_secondary">
        <property name="text">
         <string>среднее - 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>общее:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="education_general_secondary">
        <property name="text">
         <string>среднее - 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="education_general_common">
        <property name="text">
         <string>основное - 4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="education_general_primary">
        <property name="text">
         <string>начальное - 5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="education_general_unknown">
        <property name="text">
         <string>неизвестно - 6</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_12" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QRadioButton" name="employment_military">
        <property name="text">
         <string>проходит военную службу и приравненную к ней службу - 2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_31">
        <property name="text">
         <string>13. Занятость:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QRadioButton" name="employment_student">
        <property name="text">
         <string>студент(ка) - 4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="employment_retiree">
        <property name="text">
         <string>пенсионер(ка) - 3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="employment_work">
        <property name="text">
         <string>работает - 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="employment_not_work">
        <property name="text">
         <string>не работает - 5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="employment_others">
        <property name="text">
         <string>прочие - 6</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_13" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>14. Инвалидность (первичная, повторная, группа, дата)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="disability">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_14" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_33">
        <property name="text">
         <string>15. Место работы:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="work_place">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_34">
        <property name="text">
         <string>должность:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="work_post">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_15" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_35">
        <property name="text">
         <string>16. Группа крови</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="blood_group">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_36">
        <property name="text">
         <string>22. Rh-фактор</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="blood_rh">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_37">
        <property name="text">
         <string>24. Аллергические реакции</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="allegic_reactions">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
